### DISCUSSION TITLE
authentik 2023.4 Beta

### DISCUSSION THREAD

**Discussion by BeryJu** [goauthentik/authentik#5224] [Announcements]
Hello everyone,

the 2023.4 release is almost ready, so we encourage everyone to try out the beta: https://goauthentik.io/docs/installation/beta, which will get you the current build to try out the new features.

See the release notes: https://deploy-preview-5223--authentik.netlify.app/docs/releases/2023.4

**Comment by rafaelmathieu** [comment]
Thanks :D
